### DISCUSSION TITLE
Request: Report Clocks as Percent/Fraction/Ratio

### DISCUSSION THREAD

**Discussion by PorcelainMouse** [Umio-Yasuno/amdgpu_top#29] [Show and tell]
Great tool, love it.  Found your reference in `radeontop` issue thread.

I like how `radeontop` reports clocks as ratio & percent of maximums.  I think it would be cool in this tool, too.  Most stats already report that way, but this could done for a few more stats, too.  Just a suggestion.

**Comment by Umio-Yasuno** [comment]
My concern is that users may confuse each usage rate with a percentage of the maximum clock.
Higher clocks do not necessarily mean higher usage.

Also, the GPU/Memory clock plot is implemented in GUI mode.
![ss0](https://github.com/Umio-Yasuno/amdgpu_top/assets/53935716/b5886156-4a42-4d49-843e-464ec8c8a353)

**Reply by PorcelainMouse** [reply]
Oh, cool.  I haven't tried GUI mode.  Looks slick.

**Comment by Umio-Yasuno** [comment]
Would this really help?

![ss](https://github.com/Umio-Yasuno/amdgpu_top/assets/53935716/d44eebda-acc5-42bc-863e-c64712ffca1e)

**Reply by PorcelainMouse** [reply]
**I** like it, anyway. If my understanding is correct, clocks are managed based on load & temperature.  So, this representation would be helpful in understanding what is going on, internally.
